e_edu_bot/config.py

```diff
@@ -0,0 +1,123 @@
+import json
+import os
+
+
```

Comment:
Consider creating multiple `Config` classes like `DevelopmentConfig`, `TestingConfig` and `ProductiveConfig` which all extend `Config` to assign yourself to admin_client in the development szenario only.

Reply:
Changed to example data! Serveradmin need to change this in the generated config File